app/src/main/java/com/example/myfridgehome/MainActivity.kt

```diff
@@ -17,12 +17,10 @@ open class MainActivity : AppCompatActivity() {
         setContentView(R.layout.activity_main)
 
 
-
-
 //        val languages = resources.getStringArray(R.array.MenuContents)
```

Comment:
While you're here, remove commented out code entirely.  Instead of commenting out code, my suggestion is:
1) Use feature branches in Git to keep experimental work separate from master.
2) If you're removing something that used to be there, just remove it entirely.  You can always go back in history to see what used to be there.